### DISCUSSION TITLE
Bandwidth & other limits to serving through Quarto.pub?

### DISCUSSION THREAD

**Discussion by davidpassmore** [quarto-dev/quarto-cli#10965] [Q&A] [answered]
### Description

Are resource/bandwidth limits specified for serving assets (web pages, etc.) through Quarto.pub?

Thank you ..... David

**Comment by cderv** [comment, marked as answer]
We document it the callout note in our doc: https://quarto.org/docs/publishing/quarto-pub.html

> Quarto Pub sites are publicly visible, can be no larger than 100 MB and have a soft limit of 10 GB of bandwidth per month. If you want to authenticate users, host larger sites, or use a custom domain, consider using a professional web publishing service like [Netlify](https://quarto.org/docs/publishing/netlify.html) instead.
